sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -329,8 +342,34 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
         { coeThresholdDate },
       );
     }
-    coeQuery.orderBy("disbursementSchedule.coeStatus");
-    return coeQuery.getMany();
+    if (paginationOptions.searchCriteria) {
+      coeQuery.andWhere(
+        new Brackets((qb) => {
+          qb.where(
+            "CONCAT(user.firstName,  ' ', user.lastName ) Ilike :searchCriteria",
+            {
+              searchCriteria: `%${paginationOptions.searchCriteria.trim()}%`,
```

Comment:
I believe that _searchCriteria_ can be added once in the query globally using _setParameter_.

Reply:
setParameter used here